inst/stan/canham_multi_ind.stan

```diff
@@ -69,12 +69,12 @@ parameters {
   real<lower=0> ind_k[n_ind];
 
   //Species level
-  real species_max_growth_mean;
-  real<lower=0> species_max_growth_sd;
-  real species_diameter_at_max_growth_mean;
-  real<lower=0> species_diameter_at_max_growth_sd;
-  real species_k_mean;
-  real<lower=0> species_k_sd;
+  real pop_max_growth_mean;
+  real<lower=0> pop_max_growth_sd;
+  real pop_diameter_at_max_growth_mean;
```

Comment:
Soemthing to think about - will it always be a diameter? Perhaps `size` is better?

Reply:
Done.